omnigibson/tasks/behavior_task.py

```diff
@@ -97,31 +83,24 @@ def __init__(
         self.activity_conditions = None
         self.activity_initial_conditions = None
         self.activity_goal_conditions = None
-        self.activity_natural_language_goal_conditions = None
         self.ground_goal_state_options = None
-        self.instruction_order = None
-        self.feedback = None
-        self.scene_model = None
+        self.feedback = None                                                    # None or str
+        self.sampler = None                                                     # BDDLSampler
 
         # Object info
-        self.object_taxonomy = ObjectTaxonomy()
-        self.debug_object_sampling = debug_object_sampling
-        self.online_object_sampling = online_object_sampling
-        self.highlight_task_relevant_objs = highlight_task_relevant_objects
-        self.object_scope = None
-        self.object_instance_to_category = None
-        self.room_type_to_object_instance = None
-        self.non_sampleable_object_instances = None
-        self.non_sampleable_object_scope = None
-        self.non_sampleable_object_conditions = None
-        self.non_sampleable_object_scope_filtered_initial = None
-        self.object_sampling_orders = None
-        self.sampled_objects = None
-        self.sampleable_object_conditions = None
-
-        # Logic-tracking info
-        self.currently_viewed_index = None
-        self.currently_viewed_instruction = None
+        self.debug_object_sampling = debug_object_sampling                      # bool
+        self.online_object_sampling = online_object_sampling                    # bool
+        self.highlight_task_relevant_objs = highlight_task_relevant_objects     # bool
+        self.object_scope = None                                                # Maps str to BDDLEntity
+        self.object_instance_to_category = None                                 # Maps str to str
+        self.future_obj_instances = None                                        # set of str
+        self.agent_obj_idx = None                                               # int
+
+        # Info for demonstration collection
+        self.instruction_order = None                                           # np.array of int
+        self.currently_viewed_index = None                                      # int
+        self.currently_viewed_instruction = None                                # tuple of str
+        self.activity_natural_language_goal_conditions = None                   # str
```

Comment:
are these still being used?

Reply:
yes (not actively, but evidently for VR)